http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/DataprocInterpreter.scala

```diff
@@ -178,10 +178,19 @@ class DataprocInterpreter[F[_]: Parallel](
         // If we need to support 2 version of dataproc custom image, we'll update this
         dataprocImage = config.dataprocConfig.customDataprocImage
 
+        // If the cluster is configured with worker private access, then specify the
+        // `leonardo-private` network tag. This tag will be removed from the master node
+        // once the cluster is running.
+        tags = if (machineConfig.workerPrivateAccess) {
```

Comment:
where is the code that removes the tag? I only see code that we're adding additional tag on master

Reply:
It is in `DataprocRuntimeMonitor`, mentioned above